src/screens/authentication/Login/index.js

```diff
@@ -4,11 +4,13 @@ import DatabaseService from '~/databaseService'
 import * as AnalyticsService from '~/analyticsService'
 import * as AuthenticationService from '~/authenticationService'
 import Auth from '../components/Auth'
-import { isValidEmail } from '../utils'
+import { isValidEmail } from '~/mixins/form/isValidEmail'
+import { verticalScale } from 'react-native-size-matters'
```

Comment:
use `react-native-size-matters/extend`, and in this case, as it's not a standalone component, prefer to move all styling into a `styles.js` file